README.md

```diff
@@ -259,7 +259,65 @@ BundleKeys.endpoints := Map("akka-remote" -> Endpoint("tcp"))
 
 In the above, no declaration of `services` is required as akka remoting is an internal, cluster-wide TCP service.
 
-## play[23|24|25]-conductr-bundle-lib
+## play25-conductr-bundle-lib
```

Comment:
> Is it possible to create reusable text blocks in markdown, similar as templates in Twirl?

Usually this is handled by having the code referenced in "snippets" using [play-doc](https://github.com/playframework/play-doc#code-snippet-syntax), but I don't know if this project uses the same method.